app/controllers/concerns/idv_session.rb

```diff
@@ -1,17 +1,15 @@
 module IdvSession
   extend ActiveSupport::Concern
 
-  included do
-    before_action :redirect_if_sp_context_needed
-  end
+  included { before_action :redirect_if_sp_context_needed }
 
   def confirm_idv_session_started
     redirect_to idv_doc_auth_url if idv_session.applicant.blank?
   end
 
   def confirm_idv_needed
     if current_user.active_profile.blank? ||
-       decorated_session.requested_more_recent_verification? || liveness_upgrade_required?
+         decorated_session.requested_more_recent_verification? || liveness_upgrade_required?
```

Comment:
Here's another example where I think putting each condition on a separate line would be cleaner (which we weren't doing before). I wonder if that's configurable somewhere?